tools/toolchains/arm.py

```diff
@@ -206,6 +210,12 @@ def correct_scatter_shebang(self, scatter_file):
                         out.write(self.SHEBANG)
                         out.write("\n")
                         out.write("".join(lines[1:]))
+
+                # Copy headers into the build dir as well
+                headers = glob.glob(join(dirname(scatter_file), '*.h'))
+                for header in headers:
+                    copyfile(header, join(self.build_dir, basename(header)))
```

Comment:
Could you find another way to fixup the include paths? Especially because this ignores the scanning rules.

Reply:
Sure, would you mind pointing me to docs I can read up on regarding scanning rules?

Reply:
Implemented as suggested